### ISSUE TITLE
Figure out best way to have example use latest plugin code

### ISSUE THREAD

**Issue by alloy** [relay-tools/relay-compiler-language-typescript#37]
Currently the example expects a package to have been built in the root of the repo.

**Comment by sibelius** [comment]
auto release https://github.com/intuit/auto?

**Comment by zephraph** [comment]
Yeah, I can set it up. I’m not sure how that exactly relates to this original issue though.

**Comment by alloy** [comment]
Hah, good spot, my jet-lagged head didn't even catch that. I _thought_ we had an issue for the release management, but I guess we don’t even have that.
